sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -385,8 +386,24 @@ internal class RadarLocationManager(
     }
 
     fun handleLocation(location: Location?, source: RadarLocationSource) {
-        logger.d("Handling location | location = $location")
-
+        if (Radar.isTestKey()) {
+            val latency = if (location == null) -1 else Date().time - location.time
+            val standbyBucket = batteryManager.getAppStandbyBucket()
+            val batteryState = batteryManager.getBatteryState()
+            logger.d(
+                "Handling location | " +
+                        "location = $location; " +
+                        "latency = $latency; " +
+                        "standbyBucket = $standbyBucket; " +
+                        "performanceState = ${batteryState.performanceState.name}; " +
```

Comment:
given this code of destructuring `batteryState` into a string representation is repeated multiple places, do we just want to add it as a method in `RadarBatteryManager`?

Reply:
@jsani-radar ideally next version we will not be logging each individual part of the BatteryState object. the deserialization piece is a mechanism for reducing the overall log payload into a status that we can easily deserialize on the backend instead. The other sub-fields were added as a response to @tjulien's [comment](https://github.com/radarlabs/radar-sdk-android/pull/154#discussion_r779587869).